### PR TITLE
[AC-9452] 1 Add program interest fields to Program model

### PR DESCRIPTION
**Pull request by shakirandagire**
**Changes made to [AC-9452](https://masschallenge.atlassian.net/browse/AC-9452):**
- Added fields in Program Model.

**How to test:**

- On this branch on accelerate, run `make migrate`
- Login or masquerade as a user with admin privileges.
- Checkout http://localhost:8181/admin.
- Ensure that you are able to see the fields in Program as indicated in the done of [AC-9452](https://masschallenge.atlassian.net/browse/AC-9452)

[Sibling PR](https://github.com/masschallenge/accelerate/pull/3255)

### CODE REVIEW

File: accelerator/migrations/0099_update_program_model.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+            model_name='program',
+            name='hubspot_url',
+            field=models.URLField(blank=True, null=True),
+        ),
+        migrations.AddField(
+            model_name='program',
+            name='program_image',
+            field=models.ImageField(null=True, upload_to=''),
+        ),
+    ]
```
Comment (iankigen) [inline accelerator/migrations/0099_update_program_model.py:26]: EOF newline

**Review by iankigen — COMMENTED** [review]
We have lint, otherwise, looks good

File: accelerator/migrations/0099_update_program_model.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    operations = [
+        migrations.AddField(
+            model_name='program',
+            name='hubspot_url',
+            field=models.URLField(blank=True, null=True),
+        ),
+        migrations.AddField(
+            model_name='program',
+            name='program_image',
+            field=models.ImageField(null=True, upload_to=''),
```
Comment (AmosWels) [inline accelerator/migrations/0099_update_program_model.py:24]: suggestion: could specify a directory path for the upload

**Review by iankigen — COMMENTED** [review]
we have conflicting migrations

**Review by iankigen — APPROVED** [review]

File: accelerator/migrations/0101_update_program_model.py [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+            name='hubspot_url',
+            field=models.URLField(blank=True, null=True),
+        ),
+        migrations.AddField(
+            model_name='program',
+            name='program_image',
+            field=sorl.thumbnail.fields.ImageField(
+              blank=True,
+              null=True
+              upload_to='program_images'),
```
Comment (iankigen) [inline accelerator/migrations/0101_update_program_model.py:28]: syntax error

**Review by iankigen — CHANGES_REQUESTED** [review]

File: accelerator_abstract/models/base_program.py [hunk 4]
```diff
@@ -146,6 +147,15 @@ class BaseProgram(AcceleratorModel):
             AcceleratorModel.Meta.app_label, 'InnovationStage'),
         blank=True,
         related_name='programs')
+    program_image = ImageField(
+        upload_to='program_images',
+        blank=True,
+        null=True
```
Comment (iankigen) [inline accelerator_abstract/models/base_program.py:153]: This should be `null=True, blank=False`

**Review by iankigen — APPROVED** [review]

**Review by AmosWels — APPROVED** [review]
changes look good. successfully run migrations

**Pull request merged**